src/VisualStudio/Core/Test.Next/Remote/SerializationValidator.cs

```diff
@@ -215,66 +215,70 @@ internal async Task VerifySolutionStateSerializationAsync(Solution solution, Che
             SolutionStateEqual(solutionObjectFromSolution, solutionObjectFromSyncObject);
         }
 
+        private static void AssertChecksumCollectionEqual(ChecksumCollection collection1, ChecksumCollection collection2)
+        {
+            Assert.Equal(collection1.Checksum, collection2.Checksum);
+            Assert.Equal(collection1.Children.Length, collection2.Children.Length);
+
+            for (var i = 0; i < collection1.Children.Length; i++)
```

Comment:
>for (var i = 0; i < collection1.Children.Length; i++) [](http://example.com/codeflow?start=12&length=53)

Isn't there an Assert.Equal that takes in two enumerables?

Reply:
Indeed. Will fix in next pr

Reply:
done.